### ISSUE TITLE
Minor: 3.2 bar plots  - kag_y vs kag_y_train

### ISSUE THREAD

**Issue by Offroadpia** [mattharrison/effective_xgboost_book#20]
I believe that 
pd.crosstab(index=X_train['major_cs'],  columns=kag_y)
should read
pd.crosstab(index=X_train['major_cs'],  columns=kag_y_train)

Thanks

**Comment by mattharrison** [comment]
Yes, you are technically correct. However, this should never matter because
of it does our data is messed up.
